From 42ac14a780f036b2a6c5554b5dbb78a73dbc7d4a Mon Sep 17 00:00:00 2001
From: Zhu Yuqing <b48954@freescale.com>
Date: Fri, 25 Apr 2014 12:42:18 +0800
Subject: [PATCH 61/70] ENGR00303287 OTA:fix last part is cut for OTA upgarde
 description

Change the layout file and make the parameter of layout_columnSpan proper

Signed-off-by: Zhu Yuqing <b48954@freescale.com>
---
 FSLOta/res/layout/main.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/FSLOta/res/layout/main.xml b/FSLOta/res/layout/main.xml
index 820dc06..b102697 100644
--- a/FSLOta/res/layout/main.xml
+++ b/FSLOta/res/layout/main.xml
@@ -31,7 +31,7 @@
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:layout_column="2"
-            android:layout_columnSpan="5"
+            android:layout_columnSpan="3"
             android:layout_gravity="bottom|center"
             android:layout_row="1"
             android:layout_rowSpan="2" />
-- 
1.8.0

